src/components/shapes/draw.js

```diff
@@ -163,15 +168,11 @@ function drawOne(gd, index) {
         } else {
             if(gd._context.edits.shapePosition) {
                 setupDragElement(gd, path, options, index, shapeLayer, editHelpers);
+            } else {
+                if(options.editable === true) {
+                    path.style('pointer-events', 'stroke');
```

Comment:
As long as this is scoped to `options.editable === true`, can't we still allow clicking on the fill when it's mostly opaque? `>0.5` seems a fine cutoff to me. I feel like that's the behavior people will expect - most importantly when there's no stroke but even when there is as long as there's a strong fill.

Reply:
Revised in https://github.com/plotly/plotly.js/pull/4810/commits/078c007652bb4458c3d7a5ce1a61302403fc4dca.